### PR TITLE
fix options page layout

### PR DESCRIPTION
**Pull request by tusharsnn**
### Changes this Pull Request makes:

-  fix options page layout for smaller screens

### Checklist:

- [x] I have read the [Contributing Guidelines](https://github.com/immattdavison/NoMoreDomains/blob/master/CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](https://github.com/immattdavison/NoMoreDomains/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [editor config](https://github.com/immattdavison/NoMoreDomains/blob/master/.editorconfig)
- [x] I have made an issue or been assigned to one for this feature/fix [(issues)](https://github.com/immattdavison/NoMoreDomains/issues)

### CODE REVIEW

File: assets/styles/options.css [hunk 1]
```diff
@@ -176,13 +182,27 @@ body {
...
+    word-wrap: break-word;
     font-size: 1.2rem;
     font-weight: 400;
     padding: 6px 0;
+
+}
+.whitelist-ent>.whitelist-del-container {
+    grid-column: auto / span 2;
+    display: grid;
+    place-items: center;
```
Comment (immattdavison) [inline assets/styles/options.css:207]: These actions should be performed with media queries to make the page responsive.

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: assets/styles/options.css (hunk 1 above)
Comment (tusharsnn) [reply assets/styles/options.css:207]: those css lines makes it reponsive without media queries. Check and let me know if anything else is required.

Comment (immattdavison) [reply assets/styles/options.css:207]: > those css lines makes it reponsive without media queries. Check and let me know if anything else is required.

Ok - will look into it.

Comment (immattdavison) [reply assets/styles/options.css:207]: @tusharsnn I am going to reject this PR as it doesn't seem to fix the problem - it should make the item 100% width on small screens.

**Pull request closed without merging**